test.js

```diff
@@ -61,3 +61,17 @@ test('npmKeyword.count()', async t => {
 	t.is(typeof cnt1, 'number');
 	t.is(cnt2, 0);
 });
+
+test('npmKeyword.count() using an array of keywords', async t => {
+	const cnt1 = await m.count('gulpplugin');
```

Comment:
`cnt1` is not a readable variable name. Use something more verbose.

Reply:
Changed.